src/main/java/org/broadinstitute/hellbender/tools/walkers/bqsr/ApplyBQSR.java

```diff
@@ -84,6 +89,7 @@
      * created on the same input data.
      */
     @Argument(fullName=StandardArgumentDefinitions.BQSR_TABLE_LONG_NAME, shortName=StandardArgumentDefinitions.BQSR_TABLE_SHORT_NAME, doc="Input recalibration table for BQSR")
+    @WorkflowInput
```

Comment:
If you had an optional input to a tool, how would you express that in the current framework?

Reply:
The "optional-ness" of the WDL arg for the input is derived from the `@Argument` optional attribute. The "optional-ness" of the inputs companions is based on the combination of the two attributes (`@Argument` and companion), with the only special handling/override case being for a required companion on an optional arg, where the `@Argument` attribute overrides the companion attribute and results in the companion arg being optional.